proxy/src/config.rs

```diff
@@ -55,17 +55,8 @@ pub struct Config {
 
     pub pod_name: Option<String>,
     pub pod_namespace: String,
-    pub pod_zone: Option<String>,
+    pub pod_zone: String,
```

Comment:
> Along similar lines, the CONDUIT_PROXY_POD_ZONE setting hasn't been fully vetted. Lots of things in Conduit currently assume the cluster zone is cluster.local.

Would it be better to just `const`ify this for the time being?

Reply:
What does that mean? A `const` must be known at compile time, so probably can't do that.

Reply:
I mean: should we remove the `pod_zone` from the configuration and just expose it as a static str until we know how this will work

Reply:
> What does that mean? A const must be known at compile time, so probably can't do that.

It would mean removing `pod_zone` completely and just using `static POD_ZONE: &str = "cluster.local";`. I will investigate that.